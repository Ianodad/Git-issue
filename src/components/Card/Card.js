import React from 'react'

function Card({title, count, style}) {
    return (
        <div className="card mx-2 my-2" style={{...style}}>
            <div className="card-body">
                <h4 className="card-title">{title}</h4>
                <p className="card-text">{count}</p>
            </div>
        </div>
    )
}

export default Card
