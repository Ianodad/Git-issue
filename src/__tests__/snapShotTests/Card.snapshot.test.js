import React from "react";
import { create } from "react-test-renderer";
import Card from "../../components/Card/Card";


describe("Card component", () => {
  test("Matches the snapshot", () => {
    const card = create(<Card />);
    expect(card.toJSON()).toMatchSnapshot();
  });
});