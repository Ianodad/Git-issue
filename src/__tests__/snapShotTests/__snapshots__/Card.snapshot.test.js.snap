// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component Matches the snapshot 1`] = `
<div
  className="col-xl-3 col-sm-6 col-12"
>
  <div
    className="card m-0 m-2"
  >
    <div
      className="card-content"
    >
      <div
        className="card-body p-1"
      >
        <div
          className="media d-flex"
        >
          <div
            className="align-self-center"
          >
            <i
              className="icon-speech warning font-large-2 float-left"
            />
          </div>
          <div
            className="media-body text-right ml-2"
          >
            <h3
              style={
                Object {
                  "color": "#443266",
                  "fontWeight": "900",
                }
              }
            />
            <span
              data-testid="card-title"
              style={
                Object {
                  "color": "",
                }
              }
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
